dev-python/pydocstyle/pydocstyle-2.0.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit distutils-r1
+
+DESCRIPTION="static analysis tool for checking compliance with Python docstring conventions."
```

Comment:
Start with uppercase, and don't finish with a dot if it's not a proper sentence (i.e. one with a verb).

Also, please group variables as in other ebuilds.